src/utils.ts

```diff
@@ -126,6 +124,58 @@ export function equal(target: ObjectInterface, base: ObjectInterface): boolean {
 	return every(target, (v, k) => v === base[k]);
 }
 
-export function toFixed(num: number) {
-	return Math.round(num * FIXED_DIGIT) / FIXED_DIGIT;
+let lastRoundNum = -1;
+let roundNumFunc = null;
+
+export function roundNumber(num: number, roundUnit: number) {
+	// Cache for performance
+	if (!roundNumFunc || lastRoundNum !== roundUnit) {
+		roundNumFunc = getRoundFunc(roundUnit);
+		lastRoundNum = roundUnit;
+	}
+
+	return roundNumFunc(num);
+}
+
+export function roundNumbers(num: ObjectInterface<number>, roundUnit: ObjectInterface<number> | number) {
```

Comment:
@jongmoon 

I think `roundPos` is better than `getRoundNumbers`.

Reply:
@daybrush I named it for universal use.